docs/pages/guides/FormikOurValidation.example.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import Code from '../../_shared/Code';
```

Comment:
Using `Code` breaks the isolation of the demo, please use something else.

```suggestion
```

I think that not displaying the error at all would provide the best tradeoff.

Reply:
What about form state understanding? Now it is really easy to understand how this example works.

Reply:
I think that it depends on the objective. Should we expect the developer to use this demo or is it only for illustration?

Reply:
if it's 2. we will prevent developer to see the source, for instance like in https://material-ui.com/components/popover/#anchor-playground. If it's 1. we absolutely need to make it work in isolation.